server/Dockerfile

```diff
@@ -145,59 +101,67 @@ FROM "${DOCKER_HUB_PROXY}debian:bullseye-slim"
         python3-setuptools \
         python3-pip \
         # PHP Requirements
-        php \
-        php-apcu \
-        php-curl \
-        php-xml \
-        php-intl \
-        php-bcmath \
-        php-mbstring \
-        php-mysql \
-        php-redis \
-        php-gd \
-        php-fpm \
-        php-zip \
+        php7.4 \
+        php7.4-apcu \
+        php7.4-curl \
+        php7.4-xml \
+        php7.4-intl \
+        php7.4-bcmath \
+        php7.4-mbstring \
+        php7.4-mysql \
+        php7.4-redis \
+        php7.4-gd \
+        php7.4-fpm \
+        php7.4-zip \
         librdkafka1 \
         libbrotli1 \
-        libsimdjson5 \
+        libsimdjson9 \
         # Unsure we need these
         zip unzip \
         # Require for advanced an unattended configuration
         curl jq \
         && apt-get autoremove -y && apt-get clean -y && rm -rf /var/lib/apt/lists/*
 
-    # Download MISP using git in the /var/www/ directory.
-    RUN if [ ! -z ${MISP_COMMIT} ]; then \
-        git clone https://github.com/MISP/MISP.git /var/www/MISP && cd /var/www/MISP && git checkout ${MISP_COMMIT}; \
-        else git clone --branch ${MISP_TAG} --depth 1 https://github.com/MISP/MISP.git /var/www/MISP; fi
-    RUN cd /var/www/MISP; git submodule update --init --recursive .; cd /var/www/MISP/app; \
-        # Remove some old and broken links that pollute the log files
-        rm -rf /var/www/MISP/INSTALL/old
+    # Download MISP using git in the /var/www/ directory. Remove unnecessary items.
+    RUN <<-EOF
+            if [ ! -z "${MISP_COMMIT}" ]; then
+                git clone https://github.com/MISP/MISP.git /var/www/MISP && cd /var/www/MISP && git checkout "${MISP_COMMIT}"
+            else
+                git clone --branch "${MISP_TAG}" --depth 1 https://github.com/MISP/MISP.git /var/www/MISP
+            fi
+            cd /var/www/MISP || exit; git -c submodule."PyMISP".update=none submodule update --init --recursive .
+            rm -r /var/www/MISP/PyMISP
+            find /var/www/MISP/INSTALL/* ! -name 'MYSQL.sql' -type f -exec rm {} +
```

Comment:
We only need the database schema, the rest is junk

Reply:
Fair. I just want to be sure that the diagnostic page won't complain about missing files. Can you verify that?

Reply:
Not due to removal of INSTALL files, but there are errors due to removal of .git:

- Current branch… You are not on a branch, Update MISP will fail
- /var/www/MISP/.git/ORIG_HEAD…File not found

Both these are quite nonsensical in a docker environment since we don't care about updating - I don't think it's worth carrying that big load of data around for the sake of these two messages. Perhaps they could be reliably patched out of MISP at build time, or perhaps I could try to submit a patch to MISP where this is behond a container environment check.

Reply:
Look at this: https://github.com/MISP/MISP/pull/9230

Since it has been merged, looks like we can set some options to get rid of those messages

Reply:
Ah that's nice - exactly what we want - I guess it could be said to be within scope of this PR since it causes this UI issue in the first place?

Reply:
I tested these, and it fixes the version check related stuff as expected, but there's still the ORIG_HEAD message. I'm thinking I'll submit a PR to MISP to add removal of that check when self_update/version_check is set to false

Reply:
[this](https://github.com/MISP/MISP/pull/9275) PR removes the .git nag on diagnostics when merged. The view is then like this:

![Screenshot 2023-09-06 at 14 40 43](https://github.com/ostefano/docker-misp/assets/153668/6d3bd3a7-826f-4685-9c07-b5874e0c0d74)